environment-dev.yml

```
@@ -5,11 +5,20 @@ dependencies:
   - python >=3.9,<3.13
   # Don't forget to sync changes between environment.yml, environment-dev.yml, and pyproject.toml!
   # Main packages
+  - cartopy
+  - geopandas
+  - intake <2.0.0 # This should be set by xdatasets once on conda-forge
+  - leafmap
   - numpy
+  - pystac
```

Il semble avoir été enlevé de ce fichier par accident.
```suggestion
  - numpy
  - planetary-computer
  - pystac
```

---

:+1: